fusion/frame_loader.py

```diff
@@ -0,0 +1,153 @@
+# The code in this contains methods to load data and generate deformable graph for Fusion Experiments 
+
+# Library imports	
+import os
+import numpy as np
+
+# Modules
+from model import dataset
+import options as opt
+from utils import image_proc
+
+# Modules for Fusion
+from create_graph_data_using_depth import create_graph_data_using_depth
+
+class RGBDVideoLoader():
```

Comment:
Is this class for loading videos, for loading graph data, or for loading camera intrinsics?

Reply:
It is for loading everything,.source and target frames and also the graph ( compute the graph and store it if not already computed)

Reply:
Perhaps "SequenceDataLoader" would work better. The graph is semantically a separate object and should be persistent throughout the sequence, but this can be done separately. I already have a class for the graph itself in my current implementation, but everything is also pretty raw and some things shouldn't be there.